app/controllers/works_controller.rb

```diff
@@ -97,15 +97,15 @@ def work_params
                      'created(1i)', 'created(2i)', 'created(3i)',
                      'created_range(1i)', 'created_range(2i)', 'created_range(3i)',
                      'created_range(4i)', 'created_range(5i)', 'created_range(6i)',
-                     :created_edtf, :abstract, :citation, :access, :license,
+                     :abstract, :citation_auto, :citation, :default_citation,
```

Comment:
Why did `created_edtf` drop out of the list? Handled elsewhere now?

Reply:
It isn't currently used. Seemed like old cruft.

Reply:
OK, great.